src/lib/iptables.ts

```diff
@@ -0,0 +1,30 @@
+import * as Promise from 'bluebird';
+import * as _childProcess from 'child_process';
```

Comment:
`_` as a prefix has a specific meaning in typescript which is that the variable isn't used at all and that it's intentional, in order to disable the unused vars warnings